File: backend/SportsReservationAPI/Models/Team/TeamDto.cs
using SportsReservationAPI.Models.Player;

namespace SportsReservationAPI.Models.Team
{
    public class TeamDto
    {
        public int Id { get; set; }
        public string? Name { get; set; }
        public string Version { get; set; } = "";
        public string Category { get; set; } = string.Empty;
        public string? Administration { get; set; }
        public bool IsPaid { get; set; }
        public bool HasAccount { get; set; }
        public bool AccountVerified { get; set; }
        public List<PlayerDto> Players { get; set; } = new();
    }
}

File: ui/src/app/models/team-dto.ts
export interface PlayerDto {
    id: number;
    firstName: string;
    lastName: string;
    email: string;
    phoneNumber: string;
    category: string;
    outfit: string;
    volunteer: boolean;
    acceptMails: boolean;
}

export interface TeamDto {
    id: number;
    name: string;
    version: string;
    category: string;
    administration: string;
    isPaid: boolean;
    hasAccount: boolean;
    accountVerified: boolean;
    players: PlayerDto[];
}
